functions/error-logger.js

```diff
@@ -0,0 +1,30 @@
+/* eslint-disable @typescript-eslint/no-var-requires */
+const Sentry = require('@sentry/node');
+
+const { SENTRY_DSN } = process.env;
+
+let sentryInitialized = false;
+
+function initErrorLogger() {
```

Comment:
@arathjz maybe instead of calling this from the messages.js file, what about moving it to the `reportError` function?